src/routes.js

```diff
@@ -27,12 +27,14 @@ module.exports.setup = function routes(app) {
     res.setHeader('Content-Type', 'application/json');
     const lang = getHtmlLang(defined(req.params.lang, ''));
     const removeRelatedContent = defined(req.query.removeRelatedContent, false);
+    const isOembed = defined(req.query.isOembed, false);
```

Comment:
Disse to må sjåast under ett i og med at begge to skal føre til det samme fra no, at lenker åpnes i ny fane og at relatert innhold vises.

Reply:
Okei, trodde du mente at removeRelatedContent brukes andre steder, så derfor var det bedre å ikke overskrive den? Eller kan jeg altså bare overskrive funksjonen dens i `contentLinkPlugin` og `relatedContentPlugin`?

Reply:
Vi kan ikkej fjerne den fordi den er i bruk. Men den skal bytte betydning. Fra å skjule relatert innhold til å vise relatert innhold men at lenker har target _blank. isOembed er lurt å ha, slik at vi kan fase ut den andre etter kvart.